plugins/jobtype/src/azkaban/jobtype/connectors/HdfsToTeradataJob.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2012 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.jobtype.connectors;
+
+import java.io.File;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+
+import com.google.common.collect.ImmutableList;
+
+import azkaban.jobtype.HadoopJavaJob;
+import azkaban.utils.Props;
+
+
+public class HdfsToTeradataJob extends HadoopJavaJob {
+  private static final Logger logger = Logger.getLogger(HdfsToTeradataJob.class);
+
+  public HdfsToTeradataJob(String jobid, Props sysProps, Props jobProps, Logger log) throws RuntimeException {
+    super(jobid, sysProps, jobProps, log);
+    jobProps.put(TdchConstants.LIB_JARS_KEY, sysProps.get(TdchConstants.LIB_JARS_KEY));
+    TeraDataWalletInitializer.initialize(new File(getCwd()), new File(sysProps.get(TdchConstants.TD_WALLET_JAR)));
```

Comment:
CWD at this point is Azkaban executor's CWD itself which is normally "/export/apps/azkaban/azkaban-exec-server/current". Let me add some comment on it.

As long as the CWD doesn't change within JVM(executor) lifetime, the location itself actually doesn't matter as it's sole purpose is to extract jar files for TDWallet, reuse it and deletes it when JVM exits. (e.g: it can even use /tmp/ directory but I was afraid of some cron job running and delete tmp file periodically).
Let me know if you still think it's worth to make it explicit, let me see how it sets CWD and how easy to make it explicit on the plugin side.